apps/am/am-role-assignment-service/demo.yaml

```diff
@@ -13,4 +13,4 @@ spec:
         BYPASS_ORG_DROOL_RULE: true
         MAX_POOL_SIZE: 10
         ROLE_ASSIGNMENT_S2S_AUTHORISED_SERVICES: ccd_gw,am_role_assignment_service,am_org_role_mapping_service,xui_webapp,aac_manage_case_assignment,ccd_data,wa_workflow_api,wa_task_management_api,wa_task_monitor,wa_case_event_handler,iac,hmc_cft_hearing_service,ccd_case_disposer,sscs
-
+      image: hmctspublic.azurecr.io/am/role-assignment-service:pr-1839-f1dbf46-20230314102012pr image into demo #{"$imagepolicy": "flux-system:demo-am-role-assignment-service"}
```

Comment:
Is the format off this correct?  i.e. will the extra text 'pr image into demo' be ignored (i.e. there to force flux to rewrite).

```
hmctspublic.azurecr.io/am/role-assignment-service:pr-1839-f1dbf46-20230314102012 #{"$imagepolicy": "flux-system:demo-am-role-assignment-service"}
```

Reply:
corrected